src/core/instance/state.js

```diff
@@ -147,7 +147,15 @@ function initComputed (vm: Component, computed: Object) {
 
   for (const key in computed) {
     const userDef = computed[key]
-    const getter = typeof userDef === 'function' ? userDef : userDef.get
+    let getter = typeof userDef === 'function' ? userDef : userDef.get
+    if (getter === undefined) {
```

Comment:
Can you wrap the if in a non-prod if please? (`if (process.env.NODE_ENV !== 'production')`)